### PR TITLE
Display token approval message on loading

### LINKED ISSUE #1253: Pool action displayed instead of approval
When doing one of the following actions: add, remove liquidity or trade inside a pool, when approving the ocean token it says that the action is taking place instead of the approval.

For example I'm doing an approval but the following message is displayed:
Screenshot 2022-03-25 at 11 18 27

### PR DESCRIPTION
**Pull request by KatunaNorbert**
Fixes #1253.

### CODE REVIEW

**Comment by KatunaNorbert** [comment]
Not the best approach since now we have 2 states holding the `isTokenApproved` values but I don't see another way without refactoring.
Other option could be as we discussed to just remove the messages and only show a spinner.

**Review by claudiaHash — APPROVED** [review]
Yes, seems to be a fair compromise

File: src/components/@shared/TokenApproval/index.tsx [hunk 1]
```diff
@@ -44,6 +46,9 @@ export default function TokenApproval({
       setTokenApproved(
         new Decimal(allowanceValue).greaterThanOrEqualTo(new Decimal(amount))
       )
+    setIsTokenApproved(
```
Comment (kremalicious) [inline src/components/@shared/TokenApproval/index.tsx:49]: What's the difference to setTokenApproved from above?

Comment (kremalicious) [reply src/components/@shared/TokenApproval/index.tsx:49]: Saw the comment now, so far down on mobile compared to the code

**Review by EnzoVezzaro — APPROVED** [review]

**Pull request merged**